Formula/nim.rb

```diff
@@ -37,6 +39,9 @@ def install
     system "./koch", "geninstall"
     system "/bin/sh", "install.sh", prefix
 
+    system "help2man", "bin/nim", "-o", "nim.1", "-N"
+    man1.install "nim.1"
+
     target = prefix/"nim/bin"
     bin.install_symlink target/"nim"
     tools = %w[nimble nimgrep nimpretty nimsuggest]
```

Comment:
```suggestion
    tools = %w[nimble nimgrep nimpretty nimsuggest]
```
Might as well run `help2man` on these tools too.